pkgs/applications/networking/cluster/k3s/update-script.sh

```diff
@@ -19,7 +19,9 @@ LATEST_TAG_RAWFILE=${WORKDIR}/latest_tag.json
 curl --silent -f ${GITHUB_TOKEN:+-u ":$GITHUB_TOKEN"} \
     https://api.github.com/repos/k3s-io/k3s/releases > ${LATEST_TAG_RAWFILE}
 
-LATEST_TAG_NAME=$(jq 'map(.tag_name)' ${LATEST_TAG_RAWFILE} | \
+LATEST_TAG_NAME=$(cat ${LATEST_TAG_RAWFILE} | \
```

Comment:
I have used `cat` on purpose (instead of last argument of `jq`). I want the input file to be first because it's naturally easier to understand this way. I'm optimizing for being easy to read/understand instead of performance.